### PR TITLE
Added Emp Box

### LINKED ISSUE #7497: Add Emphasis box to the need help section on /firefox/unsupported-systems/
## Description

Add Emphasis box to the "need help" section on [/firefox/unsupported-systems/](https://www.mozilla.org/en-US/firefox/unsupported-systems/) once the emphasis box is available in protocol [mozilla/protocol/issues/385] as discussed in https://github.com/mozilla/bedrock/pull/7470.  

Should look something like this:

Screen Shot 2019-07-30 at 1 00 14 PM

## 🐬  Success Criteria 🐬 
- [ ] Page has an emphasis box around the get help content

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes #7497

### CODE REVIEW

File: media/css/firefox/unsupported-systems.scss [hunk 1]
```diff
@@ -29,3 +29,12 @@ $image-path: '/media/protocol/img';
 .c-help-block-title {
     @include text-display-xs;
 }
+
+.mzp-c-box-emphasis {
```
Comment (alexgibson) [inline media/css/firefox/unsupported-systems.scss]: There's no need to provide custom styling for this component. You can import the CSS using the following line right after `@import '../../protocol/css/includes/lib';`

```
@import '../../protocol/css/components/emphasis-box';
```

Comment (Patil2099) [reply media/css/firefox/unsupported-systems.scss]: @alexgibson  This file doesn't exist in the project. What should I do ?

Comment (alexgibson) [reply media/css/firefox/unsupported-systems.scss]: @Patil2099 Protocol is imported as an NPM module. It should just work

**Comment by Patil2099** [comment]
@alexgibson Is this correct now?

**Comment by alexgibson** [comment]
@Patil2099 it looks good to me, does it work for you?

**Pull request merged**